ironic/templates/compute/daemonset.yaml.j2

```diff
@@ -0,0 +1,59 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
```

Comment:
nova compute should be a DaemonSet or I would even opt to make it a PetSet (consitent hostnames).